### PR TITLE
Trezor: displayaddress

### PR DESCRIPTION
**Pull request by Sjors**
Tested with Trezor T for mainnet, testnet using legacy, p2sh-wrapped and native Segwit addressees.

### CODE REVIEW

**Comment by achow101** [comment]
Please rebase onto master now that #66 has been merged.

**Comment by Sjors** [comment]
@achow101 rebased

File: hwilib/devices/trezor.py [hunk 1]
```diff
@@ -177,8 +177,13 @@ def sign_message(self, message, keypath):
 
     # Display address of specified type on the device. Only supports single-key based addresses.
     def display_address(self, keypath, p2sh_p2wpkh, bech32):
-        raise NotImplementedError('The HardwareWalletClient base class does not '
-            'implement this method')
+      expanded_path = tools.parse_path(keypath)
+      output = self.client.get_address(
```
Comment (achow101) [inline hwilib/devices/trezor.py]: trezor 0.11.0 moved this to `trezorlib.btc.get_address`.

Comment (Sjors) [reply hwilib/devices/trezor.py]: Ok, I'll rebase.

**Comment by Sjors** [comment]
Rebased to support the new Trezor Python driver. Also added tests, though I haven't run those locally and I'm not sure if they would actually catch an error.

**Comment by achow101** [comment]
tACK 1e7ac4015d0163675d96057280fb1aa49dfdcebb

The test should at least check that the trezor api is being used correctly.

**Pull request merged**